docs/synapse/userguides/ug012_storm_ref_datamod.rst

```diff
@@ -0,0 +1,299 @@
+Storm Reference - Data Modification Operators
+=============================================
+
+The operators below can be used to modify the Synapse hypergraph by:
+
+* adding or deleting nodes
+* adding cross-reference (xref) nodes
+* setting or modifying properties on nodes
+* adding or deleting tags from nodes
+
+(**Note:** currently there is no operator for deleting properties from nodes (e.g., ``delprop()``). This is planned for a future release.)
+
+All of these operators are defined in `storm.py`__.
+
+**IMPORTANT:** Synapse does not have an "are you sure?" prompt. Caution should be used with operators that can modify Synapse data, especially when used on the output of complex queries that may modify (or delete) large numbers of nodes. It is **strongly recommended** that you validate the output of a query by first running the query on its own to ensure it returns the expected results before applying any operator that will modify that data.
```

Comment:
This is better as an admonition - probably a 'danger' or 'warning' directive.
http://docutils.sourceforge.net/docs/ref/rst/directives.html#admonitions